sensor_response_curves/resample.py

```diff
@@ -30,6 +30,6 @@ def resample_response_curves(
             bounds_error=False, fill_value=0)
     start_wv = wavelength[0]
     end_wv = wavelength[-1]
-    nsteps = int((end_wv - start_wv) // resolution + 1)
+    nsteps = round((end_wv - start_wv) / resolution) + 1
```

Comment:
```suggestion
    nsteps = int(round((end_wv - start_wv) / resolution)) + 1
```

Reply:
`round` already returns an `int`, when `ndigits` is (as here) not specified -- but if you prefer the explicit cast, then I'll happily do that

Reply:
🧙